src/App.tsx

```diff
@@ -15,71 +16,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  clearSelection = () => this.setState({ selectedGood: '' });
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  render() {
+    const { selectedGood } = this.state;
+    const hasSelection = Boolean(selectedGood !== '');
```

Comment:
```suggestion
    const hasSelection = Boolean(selectedGood);
```
<img width="793" alt="Screenshot 2023-01-30 at 13 22 26" src="https://user-images.githubusercontent.com/94292654/215832208-7e4965f7-c71c-4c9d-8b01-d23ec1801424.png">